### PR TITLE
docs: update guide for new worker refactoring

### PR DESCRIPTION
**Pull request by kairoaraujo**
online key configuration changes

----
📚 Documentation preview 📚: https://repository-service-tuf--704.org.readthedocs.build/en/704/

### CODE REVIEW

File: docs/source/guide/deployment/setup.rst [hunk 1]
```diff
@@ -124,49 +109,57 @@ Disconnected
...
-(``payload.json``) file. The :ref:`guide/deployment/setup:Bootstrap` requires the
-payload.
+(``ceremony-payload.json``) file. The :ref:`guide/deployment/setup:Bootstrap`
+requires the payload.
 
 .. note::
-    The payload (``payload.json``) contains only public data, it does not contain
-    private keys.
+    The payload (``ceremony-payload.json``) contains only public data, it does
+    not contain private keys.
```
Comment (MVrachev) [inline docs/source/guide/deployment/setup.rst:117]:
```suggestion
    not contain any private keys.
```

File: docs/source/guide/deployment/setup.rst [hunk 2]
```diff
@@ -124,49 +109,57 @@ Disconnected
...
 
-    ❯ rstuf admin ceremony -b -u
+    ❯ rstuf admi-legacy ceremony -b -u -f ceremony-payload.json --api-url https://rstuf-api-url
+    Starting online bootstrap
+    Bootstrap status: ACCEPTED (c1d2356d25784ecf90ce373dc65b05c7)
+    Bootstrap status:  STARTED
+    .Bootstrap status:  SUCCESS
+    Bootstrap completed using `ceremony-payload.json`. 🔐 🎉
+
+Alternatively, you can use curl to send the payload to the RSTUF API.
```
Comment (MVrachev) [inline docs/source/guide/deployment/setup.rst:161]:
```suggestion
Alternatively, you can use the refactored `rstuf admin` and use curl to send the payload to the RSTUF API.
```

**Review by MVrachev — COMMENTED** [review]
Have you tried following the changed instructions under
`docs/source/guide/deployment/guide/k8s/` folder to deploy Kubernetes with RSTUF instance? 
This could be a good way to validate we have not missed or removed anything.

File: docs/source/guide/deployment/setup.rst (hunk 2 above)
Comment (MVrachev) [reply docs/source/guide/deployment/setup.rst:161]: Maybe we can hint that the new admin can be used that way.

**Comment by kairoaraujo** [comment]
> Have you tried following the changed instructions under `docs/source/guide/deployment/guide/k8s/` folder to deploy Kubernetes with RSTUF instance? This could be a good way to validate we have not missed or removed anything.

Tested!

**Review by MVrachev — APPROVED** [review]

**Pull request merged**